client/src/partials/depots_management/depots_management.html

```diff
@@ -0,0 +1,121 @@
+<!-- breadcrumb header -->
+<bh-breadcrumb
+  path="DepotCtrl.bcPaths"
+  button="DepotCtrl.bcButtons">
+</bh-breadcrumb>
+
+<!-- main content -->
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+
+      <div class="col-md-6">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            {{ "DEPOT.DEPOTS_LIST" | translate }}
+          </div>
+          <table class="table table-bordered table-condensed">
+            <thead>
+              <tr>
+                <th>{{ "DEPOT.REFERENCE" | translate }}</th>
+                <th>{{ "DEPOT.NAME" | translate }}</th>
+                <th>{{ "DEPOT.IS_WAREHOUSE" | translate }}</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="depot in DepotCtrl.depotList">
+                <td>{{ depot.reference }}</td>
+                <td>{{ depot.text }}</td>
+                <td  class="text-center">
+                  <i ng-if="depot.is_warehouse" class="glyphicon glyphicon-ok-sign"></i>
+                </td>
+                <td class="text-center">
+                  <a name="depot-{{depot.text}}" class="btn btn-xs btn-default"
```

Comment:
Why use the `name` attribute on this `<a>` tag?  Is it better than using a `<a name="depot-{{ depot.id }}">`?